### PR TITLE
HiDPI scroll

### PR DESCRIPTION
**Pull request by dcnieho**
Workaround discussed in #172

### CODE REVIEW

File: src/input/scroll.c [hunk 1]
```diff
@@ -203,7 +203,16 @@ int scroll_decay(view_tile *position)
 
 int scroll_get_direction(const mouse *m)
 {
-    if (!m->is_inside_window && !m->is_touch) {
+    int is_inside_window = m->is_inside_window;
+    int width = screen_width();
+    int height = screen_height();
+    if (setting_fullscreen() && m->x < width && m->y < height)
+    {
```
Comment (bvschaik) [inline src/input/scroll.c]: Please keep braces style in line with the other code (i.e. `{` on the same line as the `if`)

File: src/platform/julius.c [hunk 2]
```diff
@@ -218,7 +218,7 @@ static void handle_mouse_button(SDL_MouseButtonEvent *event, int is_down)
 static void handle_window_event(SDL_WindowEvent *event, int *window_active)
 {
     switch (event->event) {
-        case SDL_WINDOWEVENT_ENTER:
+    case SDL_WINDOWEVENT_ENTER:
```
Comment (bvschaik) [inline src/platform/julius.c]: Keep code style uniform please

File: .gitignore [hunk 3]
```diff
@@ -10,3 +10,4 @@ res/version.rc
 res/version.txt
 res/julius.icns
 /CMakeSettings.json
+out/
```
Comment (bvschaik) [inline .gitignore]: Always end the file with a newline (see: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

**Comment by dcnieho** [comment]
ok, done!

**Pull request merged**